modules/local/vsearch_usearch_global.nf

```diff
@@ -0,0 +1,33 @@
+process VSEARCH_USEARCH_GLOBAL {
```

Comment:
This process is not used and the file can be removed, but on the other hand it doesn't do any harm either. I must have forgotten to remove it when I made an nf-core module instead.

Reply:
Thanks! Will remove it!